server/etc/pulp/server.conf

```diff
@@ -337,3 +337,26 @@
 # port: 25
 # from: no-reply@your.domain
 # enabled: false
+
+# = Lazy =
+#
+# Settings for lazy content loading.
+#
+# enabled:
+#   This controls whether redirect is enabled
+#
+# redirect_host:
+#   The host FQDN or IP to which requests are redirected.
+#
+# redirect_port:
+#   The TCP port to which requests are redirected
```

Comment:
I'm not sure if here is the best place, but we'll definitely want to make it clear somewhere that the hostname and port need to match the ones configured in the Apache reverse proxy. I can definitely see people messing this up, especially when it comes to TLS where the hostname needs to match whatever common name they have in their reverse proxy certificate.